src/App.tsx

```diff
@@ -1,22 +1,65 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import {
   HashRouter as Router,
   Redirect,
   Route,
   Switch,
 } from "react-router-dom";
+import * as wn from "webnative";
+import { FilePath } from "webnative/path";
 import AuthRoute from "./components/AuthRoute";
 import Editor from "./pages/Editor";
 import Login from "./pages/Login";
 import Posts from "./pages/Posts";
+import { Feed } from "./utils/feed";
+import { useWebnative } from "./context/webnative";
 
 function App() {
+  const { fs, username } = useWebnative();
+  const [feed, setFeed] = useState<Feed | null>();
+
+  useEffect(() => {
```

Comment:
You're absolutely right- the feed logic needs to live outside of Posts (which was intended as temporary). I'm okay with this as a next iteration, but ultimately this should be handled via a context. We're going to end up writing to the feed regularly. 

I've written this up as a separate issue: https://github.com/fission-suite/blog/issues/6